src/data/faq_de.json

```
@@ -2030,11 +2030,11 @@
             "G": [
                 {
                     "term": "Genesenen-Zertifikat",
-                    "description": "Das Genesenen-Zertifikat ist ein => digitales Zertifikat, das einer Person ausgestellt wird, die nachweislich eine COVID-19-Erkrankung überstanden und die Immunität durch einen (PCR-) Test nachgewiesen hat."
+                    "description": "Das Genesenen-Zertifikat ist ein => digitales Zertifikat, das einer Person ausgestellt wird, die nachweislich eine COVID-19-Erkrankung überstanden und die Immunität durch einen PCR-Test nachgewiesen hat."
                 },
                 {
                     "term": "Gültigkeit eines Zertifikats",

```

```suggestion
                    "term": "Gültigkeit eines Impfzertifikats",
```
The description is only exact for a vaccination certificate, not for a test certificate, nor for a recovery certificate.

Sorry that I didn't notice that before!

If somebody with more knowledge could provide more information, then maybe the entry could be extended to cover the other two types of certificate.

---

Thank you & no problem that you didn't notice it before! Done in https://github.com/corona-warn-app/cwa-website/pull/1483/commits/909d4a69b632cd585b30be3b9677732d324d79ff.